Model/AdyenInitiateTerminalApi.php

```diff
@@ -88,9 +91,9 @@ public function __construct(
         $client = $this->adyenHelper->initializeAdyenClient($this->storeId, $apiKey);
 
         //Set configurable option in M2
-        $posTimeout = $this->adyenHelper->getAdyenPosCloudConfigData('pos_timeout', $this->storeId);
-        if (!empty($posTimeout)) {
-            $client->setTimeout($posTimeout);
+        $this->timeout = $this->adyenHelper->getAdyenPosCloudConfigData('total_timeout', $this->storeId);
```

Comment:
Could we use `Config` helper's new method here to get the configuration value instead of `Data` helper?